src/components/PoolOverview.tsx

```diff
@@ -138,7 +122,9 @@ export default function PoolOverview({
                   variant="filled"
                   size="small"
                   label={chipLabel}
-                  color={isOutdated || shouldMigrate ? "secondary" : "error"}
+                  color={
+                    poolData.isGuarded || shouldMigrate ? "secondary" : "error"
```

Comment:
the `shouldMigrate` ternary is reused pretty often. Is it possible to move this into a mini private function somewhere?